style.css

```diff
@@ -1,9 +1,437 @@
+
+.section {
+    /*  padding: 0 103px 0 105px;*/
+    margin: 0 auto;
+  }
+
+
+.color {
+  background-color: #D9DCD7;
+
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  color: #1C1D21;
+}
+
+.header {
+  display: flex;
+  height: 30px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
 }
-.nav{
+
+.link_flex_container {
+  display: flex;
+  justify-content: flex-end;
+  width: 80%;
+}
+
+.logo {
+  font-size: 25px;
+}
+
+.links {
+  margin: 0px 10px;
+  text-decoration: none;
+  color: black;
+  text-transform: capitalize;
+  font-size: 25px;
+}
+
+.sign_container{
     display:flex;
-    flex-direction: row;
+    justify-content: flex-end;
+}
+.sign {
+  text-decoration: none;
+  color: black;
+  text-transform: capitalize;
+  padding-right: 115px;
+  font-size: 25px;
+}
+
+
+
+
+.section_hero {
+  padding-bottom: 84px;
+  background-color: #D9DCD7;
+}
+
+.section_container {
+  display: flex;
+  min-height: 900px;
+  justify-content: center;
+}
+
+.main_photo {
+  height: 436px;
+  width: 609px;
+}
+
+.column {
+  display: flex;
+  width: 609px;
+  align-items: flex-end;
+}
+
+.catch_phrase {
+  display: flex;
+  flex-direction: column;
+  width: 605px;
+  padding-bottom: 65px;
+
+}
+
+.invite_phrase {
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+}
+
+.catch_phrase .invite_phrase {
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
+  font-size: 96px;
+  font-weight: 400;
+
+}
+
+.photo {
+  padding-bottom: 65px;
+
+}
+
+.button_container {
+  display: flex;
+  justify-content: flex-start;
+  width: 609px;
+
+}
+
+.btn {
+  border-radius: 32px;
+  padding: 9px 30px;
+  font-size: 16px;
+  background-color: #D9DCD7;
+
+}
+
+/* TODO: order styles by sections/content */
+.second_page_container {
+  padding-bottom: 120px;
+}
+
+.section_title {
+  margin: 88px 0;
+  font-size: 36px;
+  font-weight: 400;
+  font-size: 36px;
+  text-align: center;
+  line-height: 119.5%;
+}
+
+.all_pros_container {
+  display: flex;
+  justify-content: center;
+}
+
+.high_way {
+  margin: 0px 8px;
+}
+
+
+.paragraph_header {
+  font-weight: bold;
+
+
+}
+
+.pros_photo {
+  width: 400px;
+  height: 324px;
+}
+
+.paragraph_description {
+  width: 400px;
+  display: flex;
+  justify-content: center;
+
+}
+
+.header_container {
+  display: flex;
+  justify-content: center;
+  width: 400px;
+}
+
+.container_for_paragraph {
+  height: 200px;
+  display: flex;
+  justify-content: center;
+}
+
+.text {
+  opacity: 0.8;
+  text-align: center;
+}
+
+
+
+
+.third_page_container {
+  display: flex;
+}
+
+.left_side {
+  display: flex;
+  height: 811px;
+  width: 712px;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #ECE0E0;
+}
+
+.right_side {
+  height: 811px;
+  width: 712px;
+}
+
+.mountains_pic {
+  height: 811px;
+  width: 712px;
+}
+
+.some_text {
+  width: 400px;
+  height: 220px;
+  text-align: left;
+  font-family: 'Roboto', sans-serif;
```

Comment:
no need for font-family here, it's inherited from body

Reply:
fixed